commit bfa27c5fb4b2204db7999c858941fbe48fa24e86
Author: Michael Cyr <mikecyr@us.ibm.com>
Date:   Fri Mar 17 00:48:21 2017 +0000

    scsi: ibmvscsis: Synchronize cmds at tpg_enable_store time
    
    [ Upstream commit c9b3379f60a83288a5e2f8ea75476460978689b0 ]
    
    This patch changes the way the IBM vSCSI server driver manages its
    Command/Response Queue (CRQ).  We used to register the CRQ with phyp at
    probe time.  Now we wait until tpg_enable_store.  Similarly, when
    tpg_enable_store is called to "disable" (i.e. the stored value is 0),
    we unregister the queue with phyp.
    
    One consquence to this is that we have no need for the PART_UP_WAIT_ENAB
    state, since we can't get an Init Message from the client in our CRQ if
    we're waiting to be enabled, since we haven't registered the queue yet.
    
    Signed-off-by: Michael Cyr <mikecyr@us.ibm.com>
    Signed-off-by: Bryant G. Ly <bryantly@linux.vnet.ibm.com>
    Tested-by: Steven Royer <seroyer@linux.vnet.ibm.com>
    Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>
    Signed-off-by: Sasha Levin <alexander.levin@verizon.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1420_bfa27c5fb4b2204db7999c858941fbe48fa24e86.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1420_bfa27c5fb4b2204db7999c858941fbe48fa24e86.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
